app/population/middleware/set-results.daily-population.js

```diff
@@ -0,0 +1,82 @@
+const populationService = require('../../../common/services/population')
+
+async function setResultsDailyPopulation(req, res, next) {
```

Comment:
Similar to some other comments, I think we shouldn't confused the collection concept with this part of the app so we can do away with `set-results.{thing}.js`.

I wonder if we even need to do this here too and whether we can assign the population in `set-population-id.js`. If could simplify a lot of this and we can handle some of the data structure directly in the controller.

Let's pair after standup and I can explain more what I mean.

Reply:
Resolving as this has been discussed during pairing